server/config/logger.js

```diff
@@ -1,8 +1,36 @@
 const { createLogger, transports, format } = require("winston");
 
-class Logger {
-  constructor() {}
+const getDateFormat = () => {
+  let date = new Date().toDateString();
+  let time = new Date().toLocaleTimeString();
+  return `${date} || ${time}`;
+};
 
+class Logger {
+  constructor() {
+    //The data relevant about the log.
+    this.logData = null;
```

Comment:
Not sure what this is doing?

Reply:
it is a method defined that will be showing the any data relevant from where I choose to log.